aa_stripe/serializers.py

```diff
@@ -17,14 +18,43 @@ class Meta:
         fields = ["stripe_card_id", "last4", "exp_month", "exp_year"]
 
 
+class StripeCardUpdateSerializer(ModelSerializer):
+    stripe_token = JSONField()
```

Comment:
Why is it JSONField? Did you mean stripe_response

Reply:
It is just string with token